app/routes/products.$productSlug/route.tsx

```diff
@@ -44,8 +44,31 @@ export default function ProductDetailsPage() {
         getInitialSelectedOptions(product.productOptions)
     );
 
+    const getIsOutOfStock = () => {
+        if (product.stock?.inventoryStatus === products.InventoryStatus.OUT_OF_STOCK) {
```

Comment:
in this function you didn't import `products` as `wixStoreProducts`

Reply:
but honestly I can see why you used wixStoreProducts before, because here we are like comparing product to products and it is a bit confusing.
while if we were to compare to wixStoreProducts, then it  would be less confusing.

I would even recommend importing only InventoryStatus

`import {products: {InventoryStatus}} from "..."` if that is possible. because that is the only thing we need, right ?

Reply:
`import {products: {InventoryStatus}} from "..."` this is not possible